components/apim-analytics/org.wso2.analytics.solutions.apim.analytics/widgets/APIMApiCreated/src/APIMApiCreated.jsx

```diff
@@ -0,0 +1,372 @@
+
+/*
+ *  Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ *
+ */
+
+import React from 'react';
+import Widget from '@wso2-dashboards/widget';
+import Moment from 'moment';
+import PlayCircleFilled from '@material-ui/icons/PlayCircleFilled';
+import Typography from '@material-ui/core/Typography';
+import Paper from '@material-ui/core/Paper';
+import CircularProgress from '@material-ui/core/CircularProgress';
+import Axios from 'axios';
+import {
+    addLocaleData, defineMessages, IntlProvider, FormattedMessage,
+} from 'react-intl';
+import CustomIcon from './CustomIcon';
+
+/**
+ * Language
+ * @type {string}
+ */
+const language = (navigator.languages && navigator.languages[0]) || navigator.language || navigator.userLanguage;
+
+/**
+ * Language without region code
+ */
+const languageWithoutRegionCode = language.toLowerCase().split(/[_-]+/)[0];
+
+/**
+ * Create React Component for APIM Api Created
+ * @class APIMApiCreated
+ * @extends {Widget}
+ */
+class APIMApiCreated extends Widget {
+    /**
+     * Creates an instance of APIMApiCreated.
+     * @param {any} props @inheritDoc
+     * @memberof APIMApiCreated
+     */
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            width: this.props.glContainer.width,
+            height: this.props.glContainer.height,
+            totalCount: 0,
+            weekCount: 0,
+            localeMessages: null,
+        };
+
+        this.styles = {
```

Comment:
Why this is not a `const` outside of the class?

Reply:
Have to go with this approach since we can't wrap the widget with higher order component 'withStyles'.